docs/upgrade/UPGRADE-v8.0.1-dev.md

```diff
@@ -2,5 +2,15 @@
 
 This guide contains instructions to upgrade from version v8.0.0 to v8.0.1-dev.
 
+
```

Comment:
Redundant line here

Reply:
fixed in cc3f128ebfb4f98518ed89cc94b2bec6f25793d1